pyomeca/processing/matrix.py

```diff
@@ -0,0 +1,53 @@
+from typing import Union
+
+import numpy as np
+import xarray as xr
+
+
+def abs_(array: xr.DataArray) -> xr.DataArray:
+    return np.abs(array)
+
+
+def matmul(array: xr.DataArray, other: xr.DataArray) -> xr.DataArray:
+    return array @ other
+
+
+def square(array: xr.DataArray, **kwargs) -> xr.DataArray:
+    return np.square(array, **kwargs)
+
```

Comment:
My question earlier probably goes here actually. Are these useful?

Reply:
IMHO, yes because you can do method chaining:

```python
from pyomeca import Angles
import numpy as np

a = Angles.from_random_data()
b = Angles.from_random_data()

# before
np.square(np.abs(np.matmul(a, b)))

# after
(a
  .meca.matmul(b)
  .meca.abs()
  .meca.square()
)
```